src/style.css

```diff
@@ -1 +1,92 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,400;1,500;1,700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+@media screen and (max-width: 1200px) {
+  .header {
+    max-width: 1200px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    max-width: 1024;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+}
+
+.img_link {
+  height: 40px;
+  width: 40px;
+}
+
+.logo_img {
+  display: block;
+  height: 100%;
+  width: 100%;
+}
+
+.nav {
+  display: block;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_list_item {
+  margin-left: 20px;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav_list_item:first-child {
+  margin-left: 0;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+a.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: #00acdc;
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)